llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -1861,7 +1886,15 @@ template <class ELFT> Error ELFBuilder<ELFT>::readSections(bool EnsureSymtab) {
 
       const typename ELFFile<ELFT>::Elf_Shdr *Shdr =
           Sections->begin() + RelSec->Index;
-      if (RelSec->Type == SHT_REL) {
+      if (RelSec->Type == SHT_CREL) {
+        auto Rels = ElfFile.crels(*Shdr);
```

Comment:
Would `RelsOrRelas` be a better name as it will make the meaning of first and second more obvious at the point of use?

Reply:
Agreed. Renamed to `RelsOrRelas`